src/test/resources/features/productDescription.feature

```diff
@@ -0,0 +1,36 @@
+
+@tag
+Feature: Title of your feature
+  I want to use this template for my feature file
+
+  @tag1
+  Scenario: Title of your scenario
+    Given I want to write a step with precondition
+    And some other precondition
+    When I complete action
+    And some other action
+    And yet another action
+    Then I validate the outcomes
+    And check more outcomes
+    
+     @tag2
+  Scenario: Title of your $cen@r!o
```

Comment:
Please check the spelling of scenario. plz correct it

Reply:
made changes